### PR TITLE
Feature/21908 spa configurable numpad removal

### PR DESCRIPTION
**Pull request by danwilliams**

### CODE REVIEW

**Comment by GuyHarwood** [comment]
just so everyone is aware - travis has not built this as its target branch is an unprotected branch

File: admin/services/config.service.js [hunk 1]
```diff
@@ -59,9 +60,10 @@ const configService = {
       winston.error('Failed to get access arrangements: ' + error.message)
     }
 
-    if (accessArrangementsCodes.includes('ATA')) {
-      checkOptions.audibleSounds = true
-    }
+    accessArrangementsCodes.forEach(code => {
+      if (code === 'ATA') checkOptions.audibleSounds = true
```
Comment (jon-shipley) [inline admin/services/config.service.js]: I'd prefer not testing against magic strings.   I think testing against a constant makes more sense, at least I would know what it meant.

Comment (danwilliams) [reply admin/services/config.service.js]: Definitely agree - updated to use what George has done recently now that it's in master.

File: admin/services/data-access/access-arrangements.data.service.js [hunk 2]
```diff
@@ -5,6 +5,19 @@ const sqlService = require('./sql.service')
 const accessArrangementsDataService = {}
 const accessArrangementCodes = {}
 
+/**
+ * Constants to describe the access arrangements codes
+ */
+accessArrangementsDataService.CODES = Object.freeze({
```
Comment (GeorgeChatzigiannis) [inline admin/services/data-access/access-arrangements.data.service.js]: That's indeed very useful.
@jon-shipley @danwilliams  we should follow this pattern in the other areas as well.

**Pull request merged**